### ISSUE TITLE
Download PDF: add version number to cover page

### ISSUE THREAD

**Issue by laurendoone** [Hyperobjekt/seda-map#497]
Feedback from client: can we add the version number to the report cover page?

Steps to reproduce:
1) select a location on the map
2) in location panel, click DOWNLOAD PDF button
3) on page 1 cover page, add "version 4.0" under the generation date

![Screen Shot 2021-01-22 at 10 03 42 AM](https://user-images.githubusercontent.com/76966798/105528524-ed8b1980-5c99-11eb-8477-a39be2afba4f.png)

**Comment by alegzandach** [comment]
@laurendoone should this be red?

**Comment by laurendoone** [comment]
@alegzandach nah, that was just for annotation purposes. let's keep the same font/color/style as the "generated on [date]" and "for more information, please visit edopportunity.org"
